candidate-standard/clause_7_core.adoc

```diff
@@ -1,9 +1,18 @@
+[[dependencies-overview]]
+== Dependencies on API-Common Core and Collections
+
+The OGC API-EDR standard is an extension of the OGC API-Common Part 1: Core and Part 2: Collections standards. Therefore, an implementation of API-EDR must first satisfy the appropriate Requirements Classes from API-Common.
+
+include::requirements/core/REQ_api-common.adoc[]
+
 [[rc_core-section]]
 === Overview
 
+The `Core` Requirements Class defines the requirements for locating, understanding, and accessing environmental data resources.
+
 include::requirements/requirements_class_core.adoc[]
 
-The `Core` Requirements Class defines the requirements for locating, understanding, and accessing environmental data resources. The `Core` Requirements Class is presented in five sections:
+ The `Core` Requirements Class is presented in five sections:
```

Comment:
Note that spaces in front of this line are causing some funny formatting.

Reply:
I've also seen some markup failures, but couldn't fathom.